nautobot_device_lifecycle_mgmt/forms.py

```diff
@@ -3,7 +3,7 @@
 from django import forms
 from django.db.models import Q
 
-from nautobot.dcim.models import Device, DeviceRole, DeviceType, InventoryItem, Platform, Region, Site
+from nautobot.dcim.models import Device, DeviceRole, DeviceType, InventoryItem, Platform, Region, Site, Manufacturer
```

Comment:
```suggestion
from nautobot.dcim.models import Device, DeviceRole, DeviceType, InventoryItem, Manufacturer, Platform, Region, Site
```

Reply:
Putting the recommendation here to help keep things in alphabetical order.

Reply:
Small nit all together of course.

Reply:
will address this in a future PR with code styling updates (based on isort).  Will keep this existing PR as specific feature based changes.